addons/hw_drivers/main.py

```diff
@@ -69,34 +70,71 @@ def send_alldevices(self, iot_client=None):
                     'devices': devices_list_to_send,
                 }  # Don't send distant_display to the db
             }
-            # disable certifiacte verification
+            # disable certificate verification
             urllib3.disable_warnings()
```

Comment:
this line might be irrelevant now (and we can remove urllib3 import if it is now unused)